File: frontend/src/constants/Status.js
export default {
    ATIVO: 1,
    INATIVO: 0,
    INDEFINIDO: -1,

    obterStatus(status) {
        return (status == 1 || status == "ATIVO" ) ? this.ATIVO : (status == 0 || status == "INATIVO" ) ? this.INATIVO : this.INDEFINIDO
    },
    obterNome(status) {
        return status == this.ATIVO ? "ATIVO" : status == this.INATIVO ? "INATIVO" : "INDEFINIDO"
    }
}

File: backend/src/main/java/com/itb/tcc/enums/Status.java
package com.itb.tcc.enums;

public enum Status {
    ATIVO(1),
    INATIVO(0);

    byte valor;

    public byte getValor() {
        return valor;
    }

    Status(int valor) {
        this.valor = (byte) valor;
    }
}
